server/index.js

```diff
@@ -82,9 +82,9 @@ secure.xss()
 .csrf()
 .xframe()
 .csp({
-  frame: [ editorHost, "https://www.youtube.com/embed/JecFOjD9I3k" ],
-  script: [ editorHost ],
-  connection: [ editorHost ]
+  frameSrc: [ editorHost, "https://www.youtube.com/embed/JecFOjD9I3k" ],
```

Comment:
Could I bother you once again to make sure tha if this youtube link needs to be updated ever, we only have to update it in one place?